src/pages/signup.js

```diff
@@ -0,0 +1,260 @@
+import React, { useState, useEffect, useContext } from "react"
+import { navigate } from "gatsby"
+import { Link } from "@reach/router"
+
+import { library } from "@fortawesome/fontawesome-svg-core"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { config } from "@fortawesome/fontawesome-svg-core"
+import "@fortawesome/fontawesome-svg-core/styles.css"
+
+import gear from "../assets/gear.svg"
+import PasswordStrengthMeter from "../components/passwordstrengthmeter"
+
+import { faGoogle, faGithub } from "@fortawesome/free-brands-svg-icons"
+import {auth,
+  signInWithGoogle,
+  signInWithGitHub,
+  prepareUserInformation,
+  getCurrentUser,
+  verifyEmail} from "../firebase/firebase"
+
+import { HeartwoodStateContext, HeartwoodDispatchContext } from "../state/HeartwoodContextProvider"
+
+library.add(faGoogle, faGithub)
+config.autoAddCss = false
+
+const SignUp = () => {
+  const state = useContext(HeartwoodStateContext)
```

Comment:
Codacy found an issue: ['state' is assigned a value but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5919976)